app/Console/Commands/createUser.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Classes\Pterodactyl;
+use App\Models\User;
+use Illuminate\Support\Facades\Hash;
+
+class createUser extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'make:user {--ptero_id=} {--password=}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create an admin account with the Artisan Console';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {   
+
+        $ptero_id = $this->option('ptero_id') ?? $this->ask('Please specify your Pterodactyl ID.');
+
+        $password = $this->option('password') ?? $this->ask('Please specify your password.');
```

Comment:
Can you please check if this password is at least 8 chars :)

Reply:
Do you already have Password Validation rules which I could use?

Reply:
you only have to verify if the password is at least 8 long for now :) better password validation can be added later on :)

Reply:
> Do you already have Password Validation rules which I could use?

As far as I believe we do not currently have this. Password just needs to be at least 8 characters.

Reply:
k

Reply:
Added new Commit